index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Addisalem seifu Portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <p>
+                My Logo
+            </p>
+            <img src="C:/Users/Wuro/Downloads/img/menue.png" alt="">
+        </nav>
+    </header>
+    <main>
+        <h1>I'm Addisalem am Very Happy to see you</h1>
+        <p class="details">i am a software developer by profesion and i help develop many softwares, you can go ahed and check out some of my projects in my portofolio, both mobile and web based and continuing to do so. if you need help regarding software development or have a project in mind feel free to contact me.</p>
+        <h2>LET'S CONNECT</h2>
+    </main>
+    <ul  class="pic-container">
+            <li><a href="https://twitter.com/beidework"><img src="C:/Users/Wuro/Downloads/img/twitter.png" alt="twitter icon"></a></li>
+            <li><a href="https://www.linkedin.com/in/addisalemseifu/"><img src="C:/Users/Wuro/Downloads/img/linkedin.png" alt="linkedin icon"></a></li>
+            <li><a href="https://github.com/addisalemseifu"><img src="C:/Users/Wuro/Downloads/img/github.png" alt="github icon"></a></li>
+    </ul>
```

Comment:
- [ ] For providing the semantic layout, we should try to enclose the heading part in a `<section>` tag. Please try to fix this issue.